pycloudlib/qemu/cloud.py

```diff
@@ -660,3 +660,12 @@ def clean(self) -> List[Exception]:
             exceptions.append(e)
 
         return exceptions
+
+    def version(self):
```

Comment:
@holmanb We should probably lru_cache this as it's unlikely to change mid integration test run, but it's something we don't want to recalculate any time we call it.